test/spec/openapi/route.js

```diff
@@ -7,6 +7,7 @@ const yaml = require('js-yaml')
 const fastifySwagger = require('../../../index')
 const {
   openapiOption,
+  openapiRelaiveOptions,
```

Comment:
why this change?

Reply:
Im using it in a test to, these are the options im able to reproduce the bug, so I thought this is the best way to test it